nixos/tests/bittorrent.nix

```diff
@@ -23,7 +23,6 @@ let
   transmissionConfig =
     { pkgs, ... }:
     {
-      environment.systemPackages = [ pkgs.transmission_3 ];
```

Comment:
The commit message says here 'use transmission 4' but this line is simply removed.